.github/workflows/linux-release.yml

```diff
@@ -0,0 +1,40 @@
+name: Linux release
+
+on:
+  push:
+    tags:
+      - '*'
+
+env:
+  RELEASE_ARCHIVE_NAME: linux-release
+
+jobs:
```

Comment:
Maybe someone knows how reuse jobs body code? I found composite actions, but it seems that own repository for them needed, and it too much for this case.

Reply:
I looks like Reusing workflow can fit https://docs.github.com/en/actions/using-workflows/reusing-workflows?learn=getting_started&learnProduct=actions#calling-a-reusable-workflow